packages/secret-numbers/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Wietse Wind
```

Comment:
@WietseWind Can the license be updated to maybe ISC like the other packages?

Reply:
That's fine 👍